addon/components/bread-crumbs.js

```diff
@@ -95,26 +97,34 @@ export default Component.extend({
 
       assert(`[ember-crumbly] \`route:${path}\` was not found`, route);
 
-      let breadCrumb = copy(getWithDefault(route, 'breadCrumb', {
-        title: classify(name)
-      }));
+      const multipleBreadCrumbs = route.get('breadCrumbs');
 
-      if (typeOf(breadCrumb) === 'null') {
-        return;
+      if (multipleBreadCrumbs) {
+        multipleBreadCrumbs.forEach((breadCrumb) => {
```

Comment:
@hbrysiewicz any reason for iterating here vs using `breadCrumbs.pushObjects(multipleBreadCrumbs)`